tests/selectedtests/task_mappings/test_create_task_mappings.py

```diff
@@ -77,10 +76,14 @@ def test_module_source_files_included(
         project_name = "project"
 
         after = datetime.combine(date(1, 1, 1), time(1, 1, 0))
-        before = datetime.combine(date(1, 1, 1), time(1, 3, 0))
 
         mappings = under_test.TaskMappings.create_task_mappings(
-            evg_api_mock, project_name, after, before, None, "module", None
+            evg_api_mock,
+            project_name,
+            after,
+            file_regex=None,
```

Comment:
I thought adding keyword arguments here made it a little easier to know what is being passed in, let me know if this is not the correct way to use them @dbradf